AE_navigation_guide.ipynb

```diff
@@ -32,39 +32,43 @@
+    "\n",
+    "[average_meteorological_year.ipynb](tools/average_meteorological_year.ipynb): Explore the concept and applications of an Average Meteorological Year to represent the mean climatological conditions over one year of hourly data.<br>*For example: What will an average year of hourly temperature data look like in a 2C warmer world?*\n",
     "\n",
-    "[explore_model_uncertainty.ipynb](explore_model_uncertainty.ipynb): Explore uncertainty across climate models, using projected variations in air temperature trends across different climate model simulations.  \n",
-    "*For example: When is it appropriate to take a mean across different global climate models?*\n",
+    "[timeseries_tranformations.ipynb](tools/timeseries_transformations.ipynb): Explore data transformation and analysis options for working with climate timeseries data using a graphical user interface (GUI).<br>*For example: How will the 90th percentile of daily temperature values change in mid-century compared to the historical baseline?*\n",
     "\n",
-    "### Extreme Weather Events \n",
-    "--------------------------\n",
-    "[threshold_tools_exceedance.ipynb](threshold_tools_exceedance.ipynb): Perform calculations and explore visualizations of threshold exceedance events using an interactive graphical user interface (GUI). An extension of the topics introduced in threshold_tools_basics.  \n",
-    "*For example: How will the frequency of an event where temperatures are above 110F for at least 5 hours in LA County change between now and end of century?* \n",
+    "### Collaborative Notebooks\n",
+    "-------------------------------------\n",
+    "#### Demand Forecast Unit\n",
+    "[annual_consumption_model.ipynb](collaborative/DFU/annual_consumption_model.ipynb): Perform calculations and explore visualizations of generated weather and climate information for an annual consumption model by producing heating and cooling degree days.<br>*For example: How will heating degree days in Sacramento change by mid-century?*\n",
     "\n",
-    "#### Extreme Weather Events: Work in Progress \n",
-    "[threshold_tools_basics.ipynb](work_in_progress/threshold_tools_basics.ipynb): Introduction to extreme value analysis. Demonstrates how to compute statistical values of interest related to extreme weather events.  \n",
-    "*For example: What is the return value for a 1 in 20 extreme heat event in Sacramento County at mid-century?*\n",
+    "[localization.ipynb](collaborative/DFU/localization_methodology.ipynb): Introduction to the bias-correction method for making climate projections at a location.<br>*For example: How does localization of global climate models to a weather station work?*\n",
```

Comment:
In general I suggest we use "localization" for this type of bias correction, to distinguish from the bias correction methods applied to the projections data in general. So, here "Introduction to the localization method for examining climate projections at a location"